core/example/tutorial/14_unmanaged_view/parallel_for_example.cpp

```diff
@@ -0,0 +1,137 @@
+/****************************************************************************
+ * Copyright (c) 2018-2019 by the Cabana authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the Cabana library. Cabana is distributed under a   *
+ * BSD 3-clause license. For the licensing terms see the LICENSE file in    *
+ * the top-level directory.                                                 *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include <Cabana_Core.hpp>
+#include <Cabana_Sort.hpp>
+
+#include <cassert>
+
+//---------------------------------------------------------------------------//
+// parallel for using an unmanaged view.
+//---------------------------------------------------------------------------//
+void parallelForExample()
+{
+    /* Declare general run parameters */
+    using MemorySpace = Kokkos::HostSpace;
+    using ExecutionSpace = Kokkos::DefaultHostExecutionSpace;
+    using DeviceType = Kokkos::Device<ExecutionSpace,MemorySpace>;
+    const int VectorLength = 8;
+
+    /*
+     * Declare Underlying data type
+    */
+
+    // Regular class form
+    class Data {
+        public:
+            double a[VectorLength];
+            double b[VectorLength];
```

Comment:
Maybe make on of these multidimensional to make clear the ordering of the data.

Reply:
Is there anything more complicated we want to do? That could always come in a second example?

Reply:
yeah for now lets keep this basic